pkg/storage/bucket/azure/config.go

```diff
@@ -28,7 +28,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	f.StringVar(&cfg.StorageAccountName, prefix+"azure.account-name", "", "Azure storage account name")
 	f.Var(&cfg.StorageAccountKey, prefix+"azure.account-key", "Azure storage account key")
-	f.StringVar(&cfg.ContainerName, prefix+"azure.container-name", "", "Azure storage container name")
+	f.StringVar(&cfg.ContainerName, prefix+"azure.container-name", "cortex", "Azure storage container name")
```

Comment:
Changing the default may break a user's setup if the container name is not explicitly set. Would an empty container name even be valid? :thinking:

Reply:
I'm pretty sure that this value isn't being used from here as it contradicts the default being used elsewhere and I can't see an empty string working.